### PR TITLE
csharp: update to new structure

### PR DESCRIPTION
**Pull request by ErikSchierboom**

### CODE REVIEW

File: languages/csharp/README.md [hunk 1]
```diff
@@ -1,260 +1,17 @@
-_Note: All subconcepts should be split out into separate files under the concepts directory and hyperlinked._
-
 # C&#35;
 
 C# is an object-oriented language.
```
Comment (iHiD) [inline languages/csharp/README.md]: I'd probably remove this and instead start with "Thanks for wanting to contribute to the C# track" etc. 

"C# is an object-oriented language." probably fits better in the `reference.md` file

Comment (ErikSchierboom) [reply languages/csharp/README.md]: It does. This was just placeholder content :)

**Comment by iHiD** [comment]
@ErikSchierboom - pls rebase + merge :)

**Comment by ErikSchierboom** [comment]
@iHiD Rebased. I can't merge as this is your personal repository :)

Note: please squash before merging

**Pull request merged**